Extend service invocation to non-Dapr endpoints docs (#3384)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-non-daprized-endpoints.md

---
type: docs

> title: "How-To: Invoke Non-Daprized Endpoints using HTTP"

Reviewer: ```suggestion
title: "How-To: Invoke non-Dapr endpoints using HTTP"
```

linkTitle: "How-To: Invoke Non-Daprized Endpoints"
description: "Call Non-Daprized endpoints from Dapr applications using service invocation"
weight: 2000
---

> This article demonstrates how to call a non-Daprized endpoint using a Daprized application over HTTP.

Reviewer: ```suggestion
This article demonstrates how to call a non-Dapr endpoint using a Dapr over HTTP.
```

> By using Dapr's service invocation API, you can communicate with endpoints outside of your Dapr environment. Doing so yields the following [Dapr service invocation]({{< ref service-invocation-overview.md >}}) benefits to developers:

Reviewer: ```suggestion
Using Dapr's service invocation API, you can communicate with endpoints that either use or do not use Dapr. Using Dapr to call endpoints that do not use Dapr not only provides a consistent API, but also the following [service invocation]({{< ref service-invocation-overview.md >}}) benefits:
```

> 1. Resiliency policies

Reviewer: ```suggestion
1. Ability to apply resiliency policies
```

> 2. Observability with tracing & metrics

Reviewer: ```suggestion
2. Call observability with tracing & metrics
```

> 3. Access control through scoping

Reviewer: ```suggestion
3. Security access control through scoping
```

> 4. Middleware pipeline components

Reviewer: ```suggestion
4. Ability to utilize middleware pipeline components
```

5. Service discovery
6. Authentication through the use of headers

> ## Service invocation api consistency

Reviewer: ```suggestion
## Service invocation consistency
```
